duradmin/src/main/webapp/WEB-INF/config/security-config.xml

```diff
@@ -25,6 +25,7 @@
     <intercept-url pattern="/jquery/**" access="IS_AUTHENTICATED_ANONYMOUSLY"/>
     <intercept-url pattern="/favicon.ico" access="IS_AUTHENTICATED_ANONYMOUSLY"/>
     <intercept-url pattern="/webstart/**" access="IS_AUTHENTICATED_ANONYMOUSLY"/>
+    <intercept-url pattern="/webjars/**" access="IS_AUTHENTICATED_ANONYMOUSLY"/>
```

Comment:
In the built duradmin war I'm not seeing a /webjars/ folder at the top level (like the js, jquery, images, and style folders). I do see jquery in the webjars jar files in WEB-INF/lib. Is this intercept-url for a webjars folder is needed?